editor/plugins/gdextension/cpp_template/addon/library_name.gdextension

```diff
@@ -0,0 +1,21 @@
+[configuration]
+
+entry_symbol = "__LIBRARY_NAME___library_init"
+compatibility_minimum = __GODOT_VERSION__
+
+[libraries]
+
+macos.debug = "bin/lib__LIBRARY_NAME__.macos.debug.framework"
+macos.release = "bin/lib__LIBRARY_NAME__.macos.release.framework"
+windows.debug.x86_64 = "bin/lib__LIBRARY_NAME__.windows.debug.x86_64.dll"
+windows.release.x86_64 = "bin/lib__LIBRARY_NAME__.windows.release.x86_64.dll"
+linux.debug.x86_64 = "bin/lib__LIBRARY_NAME__.linux.debug.x86_64.so"
+linux.release.x86_64 = "bin/lib__LIBRARY_NAME__.linux.release.x86_64.so"
+linux.debug.arm64 = "bin/lib__LIBRARY_NAME__.linux.debug.arm64.so"
+linux.release.arm64 = "bin/lib__LIBRARY_NAME__.linux.release.arm64.so"
+linux.debug.rv64 = "bin/lib__LIBRARY_NAME__.linux.debug.rv64.so"
+linux.release.rv64 = "bin/lib__LIBRARY_NAME__.linux.release.rv64.so"
```

Comment:
Some lib entries are missing, particularly wasm. I realize that the original example project doesn't have them, but related questions keep popping on Discord about this.

Reply:
Added Android, iOS, and web library entries.